### PR TITLE
Screen/signup

### PR DESCRIPTION
**Pull request by sanjitsarkar**
Added signup screen

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -5,8 +5,11 @@
   transform: scale(0.9);
   object-fit: contain;
 }
-
+header {
+  /* backdrop-filter: blur(10px);
+  box-shadow: 0 0 1rem var(--primary-color-6); */
```
Comment (DivyanshSareen) [inline css/style.css:10]: remove commented code

File: login.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                <li><a href="">
+                        <div class="badge-holder">
+                            <i
+                                class="fa fa-heart grid place-content-center w-12 h-12 bg-pink p-2 br-full text-light"></i>
+                            <span class="badge badge-dark">0</span>
+                        </div>
+                    </a></li>
+                <li><a href="">
+                        <img src="https://bestprofilepictures.com/wp-content/uploads/2021/04/Cool-Profile-Picture.jpg"
+                            alt="" class="avatar avatar-xsm">
```
Comment (DivyanshSareen) [inline login.html:44]: alt is empty

File: login.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                    </a></li>
+            </ul>
+        </nav>
+    </header>
+    <main class="container h-full t-5 mt-5 l-0 r-0 relative grid place-content-center text-dark w-full">
+        <form class="p-5 w-full  bg-light br-sm">
+            <label class="text-2xl mb-2 block text-center text-dark text-400">Login</label>
+            <div class="col flex-gap-1 mb-2">
+                <div class="input-group">
+                    <!-- <label for="username">Username</label> -->
```
Comment (DivyanshSareen) [inline login.html:55]: don't skip labels, screen readers can't read placeholders sometimes

**Review by DivyanshSareen — APPROVED** [review]
site looks good, ui matches the theme 👍

File: css/style.css (hunk 1 above)
Comment (sanjitsarkar) [reply css/style.css:10]: Okay I will remove those companies

File: login.html (hunk 3 above)
Comment (sanjitsarkar) [reply login.html:55]: Okay For accessibility purpose I have to readd those

File: login.html (hunk 2 above)
Comment (sanjitsarkar) [reply login.html:44]: Yes, It's needed

**Review by sanu29 — APPROVED** [review]
Approved

File: css/style.css (hunk 1 above)
Comment (sanjitsarkar) [reply css/style.css:10]: *comments

**Pull request merged**